src/View/Theme/index.html.twig

```diff
@@ -0,0 +1,104 @@
+{% extends 'layout.html.twig' %}
+
+{% block title %}
+	KNOWLEDGE
+{% endblock %}
+
+{% block navbar %}
+	<nav class="navbar navbar-expand-md bg-dark">
+		<div class="container-fluid p-0">
+			<ul
+				class="navbar-nav text-center align-items-center">
+				{# {% for subject in subjects %} #}
+				{% for i in range(0,7) %}
```

Comment:
Pourquoi faire une range sur le for ? Afficher tous les sujets ici

Reply:
un autre menu est prévu pour afficher les autres sujets et laisser de la place.